src/installer/managed/Microsoft.NET.HostModel/ComHost/ComHost.cs

```diff
@@ -19,10 +21,12 @@ public class ComHost
         /// <param name="comHostSourceFilePath">The path of Apphost template, which has the place holder</param>
         /// <param name="comHostDestinationFilePath">The destination path for desired location to place, including the file name</param>
         /// <param name="clsidmapFilePath">The path to the *.clsidmap file.</param>
+        /// <param name="typeLibraries">Resource ids for tlbs and paths to the tlb files to be embedded.</param>
         public static void Create(
             string comHostSourceFilePath,
             string comHostDestinationFilePath,
-            string clsidmapFilePath)
+            string clsidmapFilePath,
+            IReadOnlyDictionary<int, string> typeLibraries = null)
```

Comment:
Can't we just ensure we pass in an empty `IReadOnlyDictionary<K,V>`? See https://github.com/dotnet/runtime/issues/24031#issuecomment-346695827 for a way.

Reply:
I'd prefer to not add an additional dependency to Microsoft.NET.HostModel just for a default value. Since it has to run on .NET Framework as part of the SDK, adding additional dependencies is always annoying.

Reply:
That is a reasonable argument.